documentation/modules/new-migrating-virtual-machines-cli.adoc

```diff
@@ -42,8 +42,9 @@ endif::[]
 
 .Procedure
 . Create a `Secret` manifest for the source provider credentials:
-+
```

Comment:
@RichardHoch - having the indent here means, when the doc build, it *indedents* then breaks out of the indent when it enters the `ifdef` clause. 

So to fix, I propose we move the *indent* inside the `ifdef` clause